graphql/schema/types/distro.graphql

```diff
@@ -138,6 +138,7 @@ input DistroInput {
   note: String!
   plannerSettings: PlannerSettingsInput!
   provider: Provider!
+  providerAccount: String!
```

Comment:
@Kimchelly Is there any opposition to forcing the user to client to always populate this field e.g. passing in an empty string when its first set if its undefined. (I noticed a bunch of distros don't have this field set at all and according to Brian it should be required)

Reply:
I believe BrianM is mistaken, the field doesn't need to be populated since the default provider account is kernel-build, which has no associated IAM role to assume. This shouldn't be a required field, it's only required if the account is a separate account that requires assuming an explicit IAM role.